services/docker-compose.yml

```diff
@@ -42,10 +42,12 @@ services:
     image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
     init: true
     environment:
+      - BUILD_DATE=${BUILD_DATE:-1970-01-01T01:00:00Z}
```

Comment:
The fact that it still fails is a limitation of the code in [services/web/server/tests/helpers/utils_environs.py](services/web/server/tests/helpers/utils_environs.py). 

Need to fix that piece of code (had already TODOs in place!)

![image](https://user-images.githubusercontent.com/32402063/73532689-7ec80700-441d-11ea-938e-b10343e642e4.png)

Reply:
done